### PR TITLE
[docs] Describe the elementAvailabilityTimeout option

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
/cc @DevExpress/testcafe-docs

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
it seems rebase required

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -213,6 +213,14 @@ TestCafe will run fixtures whose names match the specified pattern.
 testcafe ie my-tests --fixture-grep *Page*
 ```
 
+### --element-timeout \<ms\>
+
+Specifies the amount of time, in milliseconds, allowed for a page element to become available before the test fails.
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: available for what?

**Comment by inikulin** [comment]
\r-

File: docs/articles/documentation/using-testcafe/command-line-interface.md (hunk 1 above)
Comment (VasilyStrelyaev) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: So, are you saying that *available for use* or *available for performing actions on* is better?
Or do we need to rename the whole thing from *available* to a more strict descrption like *added to DOM*, *loaded* or whatever is technically correct in this situation IDK

Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: > available for use or available for performing actions on

This.

Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: Or, even better we should define what `available` means (becomes visible and present in the DOM).

**Comment by VasilyStrelyaev** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
Hope it didn't get too sophisticated

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
lgtm

**Pull request merged**